README.md

```diff
@@ -34,27 +34,37 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+    a) Semantic HTML is extremely helpful for laying out the structure of your page in an intuitive and easy-to-read way. Get into the habit of using semantic HTML frequently; don't be lazy and just use divs everywhere for everything.
+
```

Comment:
I completely agree, accessibility is very dependent on it as well. Accessibility violations can have legal repercussions on some tech industries here is an extra resource for accessibility in web development. https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML